cypress/e2e/wallet-dropdown.test.ts

```diff
@@ -1,8 +1,8 @@
 import { getTestSelector } from '../utils'
```

Comment:
non-blocking nit: these test improvements aren't related to removing and ignoring the translation files and would preferably be in their own PR